scripts/start_herb2_xfile.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Start HERB functionality in screen sessions
```

Comment:
What does it mean to start HERB functionality? Do you mean to run this sh file in screen mode? Do we need to do that even though launcher uses tmux?

Reply:
fixed the comments.